src/Symfony/Component/HttpClient/Tests/DataCollector/HttpClientDataCollectorTest.php

```diff
@@ -24,11 +24,6 @@ public static function setUpBeforeClass(): void
         TestHttpServer::start();
     }
 
-    public static function tearDownAfterClass(): void
-    {
-        TestHttpServer::stop();
```

Comment:
one test server instance can handle all test cases, no need to start/stop it all the time